Client/Circle.cpp

```diff
@@ -1,25 +1,32 @@
 #include "stdafx.h"
 #include "Circle.h"
 
-Circle::Circle() : Shape(), radius(0) {}
+Circle::Circle() :Shape(), radius(0) { setCenter(Vector2D()); }
```

Comment:
Just initialize the center inside the bracket instead 
`vertices.reserve(1);`
`vertices.at.(0) = Vector2D();`